### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by keywordnew**
Addresses: https://github.com/nodejs/community-committee/issues/559

Guidance by: https://github.com/nodejs/community-committee/issues/271#issuecomment-471488180

### CODE REVIEW

**Comment by Trott** [comment]
There's nothing to license here code-wise so I'm not sure MIT license makes sense?

**Comment by bnb** [comment]
@trott this comes directly from the Node.js Foundation Legal Committee’s advice on the matter, linked in the OP. Here’s the direct quote:

> *What do we do for mostly writing / process repos?* > If the material is documentation intended to be provided to users, use the CC-BY-4.0. For internal community documents, use the MIT license.

**Review by bnb — COMMENTED** [review]
One additional piece of advice provided at some point was to not include the year in the copyright. I would recommend removing that. Otherwise LGTM.

**Comment by Trott** [comment]
> @Trott this comes directly from the Node.js Foundation Legal Committee’s advice on the matter, linked in the OP. Here’s the direct quote:
> 
> > *What do we do for mostly writing / process repos?* > > If the material is documentation intended to be provided to users, use the CC-BY-4.0. For internal community documents, use the MIT license.

Definitely not going to second-guess them then, that's for sure. Thanks.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,9 @@
+MIT License
+
+Copyright (c) 2020 OpenJS Foundation
```
Comment (Trott) [inline LICENSE]:
```suggestion
Copyright (c) OpenJS Foundation
```

Per @bnb's comment.

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by WaleedAshraf — APPROVED** [review]

**Review by codeekage — APPROVED** [review]

**Review by bnb — APPROVED** [review]
:shipit:

**Comment by keywordnew** [comment]
I'm not able to merge this, request assistance please! 🙏🏽

**Pull request merged**